### ISSUE TITLE
Bug: SwitchField doesn't meet WCAG AA minimum contrast ratio

### ISSUE THREAD

**Issue by reesscot** [aws-amplify/amplify-ui#1820]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Other

### How is your app built?

N/A

### Please describe your bug.

SwitchField doesn't meet minimum WCAG AA for User Interface Components
![Screen Shot 2022-05-03 at 4 45 14 PM](https://user-images.githubusercontent.com/6165315/166605285-19b88f4b-54a5-4260-be79-17faa6e3367e.png)

Current track background color is #8E939F and contrast when in off state is 1.14:1 and should be a minimum of 3:1.
![Screen Shot 2022-05-03 at 4 53 59 PM](https://user-images.githubusercontent.com/6165315/166605247-34386ae9-1ae0-4323-96c6-1a52651266b0.png)
https://webaim.org/resources/contrastchecker/

It looks like we need to add a neutral 70 color between the 60 and 80. I tried `--amplify-color-neutral-60` and it was too light. `--amplify-color-neutral-80` works but it seem visually too contrasty. @dbanksdesign do you have an objection to adding a level 70? I know it breaks from the pattern of the other color ranges.
```
    60: { value: 'hsl(210, 6%, 70%)' },
    80: { value: 'hsl(210, 10%, 40%)' },
```

### What's the expected behaviour?

Should meet WCAG contrast standards.

### Help us reproduce the bug!

Visit the docs:
https://ui.docs.amplify.aws/components/switchfield
Switch to Light mode
Notice that the switch color doesn't have much contrast.

### Code Snippet

```typescript
// Put your code below this line.

```

**Comment by hbuchel** [comment]
We should also consider dark mode and aim to meet 3:1 for that.
<img width="167" alt="Screen Shot 2022-05-05 at 11 58 21 AM" src="https://user-images.githubusercontent.com/376920/166965897-36f539fb-dfab-45e5-835b-6a52d79bc6e2.png">

<img width="918" alt="Screen Shot 2022-05-05 at 12 10 14 PM" src="https://user-images.githubusercontent.com/376920/166966222-5ba70f9a-881a-4231-bfe4-1aea1eb838fe.png">

Can we also apply the 3:1 fix to the radio input control?
